frontend/src/routes/AdminRoute.tsx

```diff
@@ -16,9 +16,11 @@ const AdminRoute: React.FC<Props> = ({ children }: Props) => {
   const [error, setError] = useState(false);
 
   useEffect(() => {
-    if (currentUser === null) {
-      setError(true);
-      return;
+    if (user) {
+      if (currentUser === null) {
```

Comment:
`if (!currentUser || !user) {` I think this is readable :) then you can merge it! Thanks!

Reply:
<img width="1280" alt="Screenshot 2023-10-19 192246" src="https://github.com/HackYourFuture-CPH/class-25-ironside/assets/33804102/173f9a14-f333-4932-93f0-d9a7f52afb54">

is doesn't work :face_with_diagonal_mouth:

Reply:
is that an error page? If so, it works well

Reply:
oh ok, now I can see that I am having issues because I am missing some ADMIN env keys... 

 
```suggestion
 type: 'service_account',
  project_id: process.env.ADMIN_PROJECT_ID,
  private_key_id: process.env.ADMIN_PRIVATE_KEY_ID,
  private_key: process.env.ADMIN_PRIVATE_KEY?.replace(/\\n/g, '\n'),
  client_email: process.env.ADMIN_CLIENT_EMAIL,
  client_id: process.env.ADMIN_CLIENT_ID,
  auth_uri: 'https://accounts.google.com/o/oauth2/auth',
  token_uri: 'https://oauth2.googleapis.com/token',
  auth_provider_x509_cert_url: 'https://www.googleapis.com/oauth2/v1/certs',
  client_x509_cert_url: process.env.ADMIN_CLIENT_CERT_URL,
  universe_domain: 'googleapis.com',
};
```